lib/load.php

```diff
@@ -70,18 +65,6 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/compat/plugin/edit-site-routes-backwards-compat.php';
 require __DIR__ . '/compat/plugin/fonts.php';
 
-// The Token Map was created during 6.6 in order to support the HTML API. It must be loaded before it.
-require __DIR__ . '/compat/wordpress-6.6/class-gutenberg-token-map-6-6.php';
-require __DIR__ . '/compat/wordpress-6.7/class-gutenberg-token-map-6-7.php';
```

Comment:
This is removing a require from a file in `6.7` compat folder. Do you know if that's expected?

Reply:
Good catch. The WP 6.7 files shouldn't be removed here.

Reply:
Ah, good catch indeed! Oversight on my part, I'll add it back 😄

Reply:
f602432a615